db/Cassandra/schema.cql

```diff
@@ -0,0 +1,34 @@
+CREATE KEYSPACE ReviewsDB WITH replication = {
+  'class':'SimpleStrategy',
+  'replication_factor': 3
```

Comment:
if you want to save HD space, you can replicate only 1 time

Reply:
thanks!